set.mm

```diff
@@ -19821,27 +19831,35 @@ Converse of the inference rule of (universal) generalization ~ ax-gen .
     sbiedw.1 $e |- F/ x ph $.
     sbiedw.2 $e |- ( ph -> F/ x ch ) $.
     sbiedw.3 $e |- ( ph -> ( x = y -> ( ps <-> ch ) ) ) $.
-    $( Version of ~ sbied with a disjoint variable condition, which does not
-       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    $( Version of ~ sbied with a disjoint variable condition, requiring fewer
+       axioms.  (Contributed by Gino Giotto, 10-Jan-2024.)  Avoid ~ ax-10 .
+       (Revised by Wolf Lammen, 28-Jan-2024.) $)
     sbiedw $p |- ( ph -> ( [ y / x ] ps <-> ch ) ) $=
+      ( wsb wi sbrimv nfim1 weq wb com12 pm5.74d sbiev bitr3i pm5.74ri ) ABDEIZ
+      CATJABJZDEIACJZABDEFKUAUBDEACDFGLDEMZABCAUCBCNHOPQRS $.
+
+    $( Obsolete version of ~ sbiedw as of 28-Jan-2024.  (Contributed by Gino
+       Giotto, 10-Jan-2024.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    sbiedwOLD $p |- ( ph -> ( [ y / x ] ps <-> ch ) ) $=
       ( wsb wi sbrim nfim1 weq wb com12 pm5.74d sbiev bitr3i pm5.74ri ) ABDEIZC
       ATJABJZDEIACJZABDEFKUAUBDEACDFGLDEMZABCAUCBCNHOPQRS $.
   $}
 
   ${
     $d x ph $.  $d x ch $.  $d x y $.
     sbiedvw.1 $e |- ( ( ph /\ x = y ) -> ( ps <-> ch ) ) $.
-    $( Version of ~ sbiedv with a disjoint variable condition, which does not
-       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    $( Version of ~ sbiedv with a disjoint variable condition, requiring fewer
+       axioms.  (Contributed by Gino Giotto, 10-Jan-2024.) $)
     sbiedvw $p |- ( ph -> ( [ y / x ] ps <-> ch ) ) $=
       ( nfv nfvd weq wb ex sbiedw ) ABCDEADGACDHADEIBCJFKL $.
```

Comment:
Here you can also remove ax-12 as well, by lengthening the proof:
<pre>
    $( Version of ~ sbiedv with a disjoint variable condition, requiring fewer
       axioms.  (Contributed by Gino Giotto, 10-Jan-2024.) $)
    sbiedvw $p |- ( ph -> ( [ y / x ] ps <-> ch ) ) $=
      ( wsb wi weq wal sb6 imbi2i bi2.04 albii 19.21v 3bitr2i wb expcom pm5.74d
      sbievw pm5.74ri ) ABDEGZCAUBHADEIZBHZDJZHZABHZDEGZACHZUBUEABDEKLUHUCUGHZD
      JAUDHZDJUFUGDEKUKUJDAUCBMNAUDDOPUGUIDEUCABCAUCBCQFRSTPUA $.
</pre>
Indeed I did not check for further improvements outside ax-13, I only checked than my *w versions wouldn't require more axioms than their original counterparts. Actually this is good news: it means that this refactoring not only will diminish ax-13 usage, but it will probably bring down other auxiliary axioms along the way.

Reply:
This calls for an additional sbrimvw with $d x y $. and $d x ph $.  conditions.

Reply:
Both ways are fine for me. I'll let you decide.

Reply:
And sbiedvw can then be moved several sections earlier.

Probably that the refactoring may also help remove some ax-11 usages, as I noticed a while ago (see "20.15.4.12  Removing dependencies on ax-13 (and ax-11)" and its comment

> It is also possible to remove dependencies on [ax-11](https://us.metamath.org/mpeuni/ax-11.html), typically by replacing a nonfree hypothesis with a disjoint variable condition (see [cbv3v2](https://us.metamath.org/mpeuni/cbv3v2.html) and following theorems).

Reply:
sbiedw is moved close to sb6 now.